src/lib/connectors/repos/licence-agreements.js

```diff
@@ -54,8 +56,8 @@ const update = async (id, changes) => {
  * Deletes a licence agreement by
  * @param {String} licenceAgreementId
  */
-const deleteOne = async licenceAgreementId =>
-  helpers.deleteOne(LicenceAgreement, 'licenceAgreementId', licenceAgreementId);
+const softDeleteOne = async licenceAgreementId =>
+  helpers.update(LicenceAgreement, 'licenceAgreementId', licenceAgreementId, { dateDeleted: moment() });
```

Comment:
Purely advisory: This is fine but I THINK you can simply pass `new Date()` without needing to import `moment`.